src/ctap/data_formats.rs

```diff
@@ -631,26 +631,39 @@ const ES256_ALGORITHM: i64 = -7;
 const EC2_KEY_TYPE: i64 = 2;
 const P_256_CURVE: i64 = 1;
 
+impl TryFrom<cbor::Value> for CoseKey {
```

Comment:
I changed the whole `CoseKey` data type. This PR was initially only about the crypto library changes. But this seems to be a good time to tackle one of my TODOs to implement a proper `CoseKey` data format since it's usage won't change much anymore.